styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -71,6 +71,11 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+@mixin center-block {
```

Comment:
So now that you're no longer using this mixin, we probably don't need to include it in this PR.  If we want to create this mixin in the future when we actually use it, that seems okay, but for now, I'd say scrap it.